kategory-core/src/main/kotlin/kategory/typeclasses/ContinuationUtils.kt

```diff
@@ -15,7 +15,7 @@ private var <T> Continuation<T>.completion: Continuation<*>?
     get() = completionField.get(this) as Continuation<*>
     set(value) = completionField.set(this@completion, value)
 
-internal var <T> Continuation<T>.stackLabels: List<Any>
+var <T> Continuation<T>.stackLabels: List<Any>
```

Comment:
Is this required? This is a dangerous API to expose to the public inside `binding`. I'd rather have the code duplicated

Reply:
It's going to be used by reimplementations of binding done for cancellable effects. I'll copypaste it then.